.semaphore/pre-release.yml

```diff
@@ -22,8 +22,18 @@ blocks:
       jobs:
         - name: "Publish to NPM"
           commands:
+            - git checkout -b pre-release-$SEMAPHORE_PIPELINE_ID
+            - git push origin pre-release-$SEMAPHORE_PIPELINE_ID --no-verify
             - nvm use
             - npx lerna bootstrap
             - yarn
-            - yarn lerna version prerelease --yes
+            - yarn lerna version --conventional-commits --conventional-prerelease --no-changelog --no-push --yes
```

Comment:
Using `--conventional-commits --conventional-prerelease` will automatically assign the correct alpha version to packages.
They're using this preset by default: https://github.com/conventional-changelog/conventional-changelog/tree/master/packages/conventional-changelog-angular I didn't find any problem with it

Reply:
Adding `--no-changelog` because we currently have our own changelog generator, they might conflict with each other. We can revisit this issue once we resolve auto versioning.

Reply:
Adding `--no-push` just for the convenience of the CI, and we will push everything (including tags) at line https://github.com/acl-services/paprika/pull/928/files#diff-c9c651f685e154b266bdbb551f05c643a0a41991154dac0413c2b8f63b19f7a0R37